CI breaks due to not being able to install shiny.react

CI breaks due to not being able to install `shiny.react`

## Example application or steps to reproduce the problem

Push to repository

### Expected behavior

CI successfully passing

### Actual behavior

In `shiny.fluent JS test` workflow `Install dependencies` step fails with error:

> Run remotes::install_github("Appsilon/shiny.react")
> Using bundled GitHub PAT. Please add your own PAT to the env var `GITHUB_PAT`
> Error: Error: Failed to install 'unknown package' from GitHub:
>   cannot open URL 'https://api.github.com/repos/Appsilon/shiny.react/contents/DESCRIPTION?ref=HEAD'
> Execution halted
> Error: Process completed with exit code 1.

--- a/.github/workflows/shiny-fluent-test.yaml
+++ b/.github/workflows/shiny-fluent-test.yaml
@@ -17,6 +17,7 @@ jobs:
       R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
       RSPM: ${{ matrix.config.rspm }}
       ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
       - name: Check out Git repository

